frontend/playwright/ui/specs/colorpicker.spec.js

```diff
@@ -0,0 +1,22 @@
+import { test, expect } from "@playwright/test";
+import { WorkspacePage } from "../pages/WorkspacePage";
+
+test.beforeEach(async ({ page }) => {
+  await WorkspacePage.init(page);
+});
+
+// Fix for https://tree.taiga.io/project/penpot/issue/7549
+test("Bug 7549 - User clicks on color swatch to display the color picker next to it", async ({ page }) => {
+  const workspacePage = new WorkspacePage(page);
+  await workspacePage.setupEmptyFile(page);
+
+  await workspacePage.goToWorkspace();
+  const swatch = workspacePage.page.getByRole("button", { name: "E8E9EA" });
```

Comment:
Just a commet/oppinion, nothing blocking for merging this PR: accesing `page` attribute on `workspacePage` object, looks redundant. I mean the naming looks confusing, is a page inside a wokspacePage?

Reply:
`page` is a Playwright `page` object, whereas `WorkspacePage` is a Page Object Model (POM). Agreed that the naming is confusing  and that probably a better name would be `WorkspacePOM` https://playwright.dev/docs/pom, but it's a pattern/convention in Playwright